templates/builders_user_data.tpl

```diff
@@ -25,6 +28,9 @@ add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(
 apt-get update
 apt-get -y install docker-ce=17.06.0~ce-0~ubuntu cgmanager
 
+sudo echo 'export http_proxy="${http_proxy}"' >> /etc/default/docker
```

Comment:
Certain calls for NPM and Yarn will require HTTPS. As such, this will need to also have:
`sudo echo 'export https_proxy="${http_proxy}"' >> /etc/default/docker`

Reply:
Done. Thanks @Aghassi!